contentcuration/kolibri_content/models.py

```diff
@@ -108,6 +108,9 @@ class ContentNode(MPTTModel):
     # Added legacy fields
     license = models.ForeignKey('License', null=True, blank=True)
 
+    # A JSON Dictionary of properties to configure loading, rendering, etc. the file
+    options = JSONField(default={})
```

Comment:
Noting that Kolibri uses a sqlite database, so not sure if this is compatible

Reply:
It's using the `jsonfield` library `JSONField` which is what Kolibri uses too, so should be ok.